app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/securities/check_your_answers.scala.html

```diff
@@ -89,7 +89,7 @@
             value = Value(Text(exportDeclarationMrn.value)),
             actions = Some(Actions(
                 items = Seq(ActionItem(
-                    href = routes.EnterExportMovementReferenceNumberController.show().url,
+                    href = routes.EnterExportMovementReferenceNumberMultipleController.show().url,
```

Comment:
We should route to EnterExportMovementReferenceNumberController if the methodOfDisposal is single shipment, and to EnterExportMovementReferenceNumberMultipleController if the methodOfDisposal is multiple shipments...

Reply:
Ignore the above, Anurag has agreed that we can use the same route for both pages so long as the content changes based on the methodOfDisposal